src/main/java/com/example/java/maven/gameBlackJack/Croupier.java

```diff
@@ -0,0 +1,28 @@
+package com.example.java.maven.gameBlackJack;
+
+import java.util.ArrayList;
+
+public class Croupier {
+    String name;
```

Comment:
private final and getters where required

Reply:
do it for all fields in all classes
https://www.tutorialspoint.com/java/java_encapsulation.htm
https://www.geeksforgeeks.org/encapsulation-in-java/

Reply:
Done